### DISCUSSION TITLE
How to get parameters in a GDSFactory generated GDS

### DISCUSSION THREAD

**Discussion by tmerelle** [gdsfactory/gdsfactory#2912] [Q&A] [answered]
Dear Joaquin, users community,

I am trying to extract instanciated objects parameters from a GDS generated using GDSFactory.
That is easy to do for the building blocks of the PDK that I use, as their parameters are included into the cell name.
But these parameters are not appearing in the instance name for routing components such as bend_euler (gf.components.bend_euler) or straight (gf.components.straight). 
Are these parameters stored somewhere else in the GDS?

Thank you in advance for your help,

Thomas

**Comment by tvt173** [comment, marked as answer]
assuming you are using gdsfactory version 8, yes... in klayout you can right click on the cell -> `User Properties`
![image](https://github.com/gdsfactory/gdsfactory/assets/3070522/9d13d310-4387-48a0-99be-a00e1eb1d8f1)

then click on the `Meta Info` tab, and you should see all your properties listed as `*kfactory:settings:<NAME>`
![image](https://github.com/gdsfactory/gdsfactory/assets/3070522/0c6e569f-d9a8-4555-baa6-4e197344209b)

in earlier versions of gdsfactory however, this was not possible

**Reply by tmerelle** [reply]
Hey Troy,

Thanks!
I am currently using GDSFactory 7 but will soon migrate to 8.
Good to know thanks to you that the info will be there.

Best,

Thomas
